nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -267,15 +267,10 @@ geometry_msgs::msg::TwistStamped RegulatedPurePursuitController::computeVelocity
   // Find look ahead distance and point on path and publish
   double lookahead_dist = getLookAheadDistance(speed);
 
-  // Check for reverse driving
-  if (allow_reversing_) {
```

Comment:
Why remove the allow_reversing function? That is necessary so that we only do this if its appropriate to be rotating in place due to a sudden change in pose, that shouldn't normally be turned on unless letting the robot move backwards is allowable by that parameterization

Reply:
Again, makes sense thanks for the clarification.